scripts/build/docker/run-command.sh

```diff
@@ -82,6 +82,8 @@ fi
 # The `-t` and TERM setup is needed to display coloured output.
 docker run -t \
   --env "TERM=xterm-256color" \
+  --env "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
```

Comment:
@WasabiFan As @lurch pointed out, the environment variables are not accessible from pull requests originating from forks: https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings